src/components/Settings/SaveSearchItem.vue

```diff
@@ -24,7 +24,7 @@
 				:href="savedSearch.url"
 				class="tw-mr-2 tw-mb-2 md:tw-mb-0"
 			>
-				View Loans
+				View {{ savedSearch.loanSearchCriteria.filters.loanIds }} Loans
```

Comment:
Looks like this needs to be the `totalCount` data property defined below.

Reply:
@tiarascoleman Could you please remove this for now and just leave `View loans` in place. I'll bring it back once I get the loan query going.